core/src/main/java/com/linecorp/armeria/server/ServiceConfig.java

```diff
@@ -392,6 +398,10 @@ public Path multipartUploadsLocation() {
         return multipartUploadsLocation;
     }
 
+    ServiceErrorHandler serviceErrorHandler() {
```

Comment:
Should we drop `service` prefix?
```suggestion
    ServiceErrorHandler errorHandler() {
```

Reply:
Okay.
It seems the method is not called by any places, so I'll delete it.